src/components/editor/ResourceTemplate.jsx

```diff
@@ -20,9 +20,11 @@ class ResourceTemplate extends Component {
     }
   }
 
-  // Called immediately after the component is mounted
+  // Called immediately after the component is rendered for the first time
```

Comment:
someone wise has conditioned me away from comments and tests that are about "how it works" - and this comment is precisely how "componentDidMount()" works.